.github/workflows/pr_test.yml

```diff
@@ -208,8 +233,23 @@ jobs:
           HEADLESS: true
           TAKE_SCREENSHOT_AFTER_FAIL: true
 
-      - run: echo "MY_COMMAND=$( echo -e '${{ matrix.TEST_COMMAND }}' | tr  ':' '-'  )" >> $GITHUB_ENV
-        if: ${{ always() }}
+      - name: Run tests develop
+        if: ${{ env.BASE_BRANCH == 'develop' }}
+        id: my_test
```

Comment:
I think this is the produce of a copy/paste as the id seems pretty generic, could you give it a better name ?

Reply:
```suggestion
```

Reply:
Fixed